shelldio.sh

```diff
@@ -127,6 +127,22 @@ remove_station() {
 	fi
 }
 
+mpv_msg() {
+	if grep debian /etc/os-release &> /dev/null; then
+		echo "Τρέξτε 'sudo apt install mpv' για να εγκαταστήσετε τον player"
+	elif grep fedora /etc/os-release &> /dev/null; then
+		echo "Τρέξτε 'sudo dnf -y install mpv' για να εγκαταστήσετε τον player"
+	elif grep suse /etc/os-release &> /dev/null; then
+		echo "Τρέξτε 'sudo zypper in mpv' για να εγκαταστήσετε τον player"
+	elif grep centos /etc/os-release &> /dev/null; then
+		echo "Τρέξτε 'sudo yum -y install mpv' για να εγκαταστήσετε τον player"
+	elif uname -a | grep Darwin &> /dev/null; then
+		echo "Τρέξτε 'sudo brew install mpv' για να εγκαταστήσετε τον player"
```

Comment:
```suggestion
		echo "Τρέξτε 'brew install mpv' για να εγκαταστήσετε τον player"
```

To Brew δεν επιτρέπει sudo χρήση.
Εννοώ ότι το αποτέλεσμα είναι literally:

```
Error: Running Homebrew as root is extremely dangerous and no longer supported.
As Homebrew does not drop privileges on installation you would be giving all
build scripts full access to your system.
```